packages/relay/src/lib/constants.ts

```diff
@@ -53,4 +53,5 @@ export default {
     TX_DEFAULT_GAS: 400_000,
     TX_CREATE_EXTRA: 32_000,
     TX_DATA_ZERO_COST: 4,
+    REQUEST_ID_STRING: `Request ID`
```

Comment:
If you add `: ` to this constant you save the need to add them in every other log line you've added them to.

```suggestion
    REQUEST_ID_STRING: `Request ID: `
```